kolibri/core/discovery/test/test_network_search.py

```diff
@@ -46,6 +47,7 @@ def test_update_instance(self, mock_enqueue):
         mock_enqueue.assert_called_once_with(
             job_id="9e89d3ea5256721c9cd631eac36feafe",
             args=(self.mock_broadcast.id, self.instance.to_dict()),
+            priority=Priority.HIGH,
         )
 
```

Comment:
@vkWeb could you add some additional tests, primarily for the priority method

Reply:
Yup will add sir

Reply:
Tests added